### PR TITLE
Example dockerfile for the OGL integration tests with OBR

### PR DESCRIPTION
**Pull request by greole**

### CODE REVIEW

File: docker/ogl_obr.dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                                                              
+# update python environment                    
+RUN python3.10 -m pip install pip --upgrade            
+                                                       
+# Clone dependencies                                   
+RUN cd $HOME && \                                      
+    git clone https://github.com/hpsim/OGL.git && \    
+    git clone https://github.com/hpsim/OBR.git && \    
+    pip install ./OBR                                  
+
```
Comment (FoamScience) [inline docker/ogl_obr.dockerfile:25]: Maybe do not include anything from this point on in the image.
The rest can be mentioned in the README as a three-steps introduction to what commands reviewers are expected to run

File: docker/ogl_obr.dockerfile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# cases and variations as defined in the integration.yaml file    
+RUN mkdir -p $HOME/OGL_integration_tests &&  cd $_ && \           
+    source /usr/lib/openfoam/openfoam2206/etc/bashrc && \     
+    obr init --config $HOME/OGL/test/integration.yaml && \    
+    obr run -o generate                                       
+                           
+    
+# Execute all unit test cases                                                                                                                              
+# this might take a while since we set the debug flag    
+RUN obr run -o runParallelSolver
```
Comment (FoamScience) [inline docker/ogl_obr.dockerfile]: I get `ERROR: failed to solve: process "/bin/bash -c obr run -o runParallelSolver" did not complete successfully: exit code: 1` here; forgot to cd somewhere?

File: docker/ogl_obr.dockerfile (hunk 1 above)
Comment (FoamScience) [reply docker/ogl_obr.dockerfile:25]: The image is supposed to provide the software and the environment, reviewers can do whatever they want.

File: docker/ogl_obr.dockerfile (hunk 2 above)
Comment (FoamScience) [reply docker/ogl_obr.dockerfile]: `LookupError: Unable to find project at path '/home/openfoam'.`

**Review by FoamScience — CHANGES_REQUESTED** [review]

File: docker/ogl_obr.dockerfile (hunk 1 above)
Comment (FoamScience) [reply docker/ogl_obr.dockerfile:25]: well the actual cutting line should be after all the cmakes and inits; so maybe line `RUN obr run -o runParallelSolver`?

File: docker/ogl_obr.dockerfile (hunk 2 above)
Comment (greole) [reply docker/ogl_obr.dockerfile]: Yes, that is true. I need to finish that up. I was unsure whether to include that as a run step or let the user/reviewer do it manually. But I think i'll do both.

File: docker/ogl_obr.dockerfile (hunk 1 above)
Comment (greole) [reply docker/ogl_obr.dockerfile:25]: The reason is that the `RUN obr run -o runParallelSolver` takes very long, for two reasons:
- The runs are usually done concurrently, on different runners, here they are run in serial.
- A debug flag is set, which lets the application wait some seconds for a debugger to attach.

I'll include the `runParallelSolver` and mention that there is the option to rerun the test cases.

**Pull request merged**